packages/mds-policy-service/package.json

```diff
@@ -27,6 +27,7 @@
     "@mds-core/mds-rpc-common": "0.1.4",
     "@mds-core/mds-schema-validators": "0.1.6",
     "@mds-core/mds-service-helpers": "0.1.4",
+    "@mds-core/mds-test-data": "0.1.30",
```

Comment:
Should be a `devDependency`. Normally I'd put this as a nit, but I'm gonna say it's a blocker here because `mds-test-data` has a ton of stuff in it that'll blow up bundle size (lots of big geojson blobs)